Add  hint for unknown node type

tplevko opened the pull request:
fix for https://github.com/KaotoIO/kaoto-next/issues/803



![Screenshot from 2024-02-06 14-55-50](https://github.com/KaotoIO/kaoto-next/assets/4180208/417d5f64-3aeb-462d-b7a6-9817bee50286)

lordrip commented on packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx:
```diff
@@ -88,9 +100,25 @@ export const CanvasForm: FunctionComponent<CanvasFormProps> = (props) => {
-          <CodeBlock>
-            <CodeBlockCode>{stringify(model)}</CodeBlockCode>
-          </CodeBlock>
+          <Card>
+            <CardHeader>
+              <CardTitle>Node source</CardTitle>
+            </CardHeader>
+            <CardBody>
+              <CodeBlock>
+                <CodeBlockCode>{stringify(model)}</CodeBlockCode>
+              </CodeBlock>
+            </CardBody>
+            <CardFooter>
+              <Hint>
+                <HintTitle>Node is not a known type</HintTitle>
+                <HintBody>
+                  Configuration properties are not available when the node is not a known type. Delete, or replace this
+                  node with valid node.
+                </HintBody>
+              </Hint>
+            </CardFooter>
+          </Card>
```
I think this component is starting to grow too big to have here :), maybe it's time to move it to a dedicated file.

tplevko replied:
sure - moved to separate file

lhein commented:
I'd rephrase the warning message a bit:

**Unknow node type**

The configuration for an unknown node cannot be changed in the configuration form. Please switch to the source code and correct the node type. Another option is to replace the step in the graphical editor but in that case you will probably lose the existing configuration.

tplevko commented:
Updated - moved to separate file, updated the `Hint` to `Alert`, with variant `warning`. Also changed the message.

With updates:
![Screenshot from 2024-02-07 10-01-21](https://github.com/KaotoIO/kaoto-next/assets/4180208/c37994f8-ec0f-44ea-8151-d56a1ba4d4fa)

lhein commented:
you did not change the headline, did you?

tplevko commented:
> you did not change the headline, did you?

Missed that the headline was also changed - Updated.

lordrip reviewed (approved):
This is a good enhancement @tplevko , thanks 💪
